plugins/modules/keycloak_user_federation.py

```diff
@@ -1018,7 +1019,7 @@ def main():
                     kc.create_component(mapper, realm)
 
             after_comp = kc.get_component(cid, realm)
-            after_comp['mappers'] = kc.get_components(urlencode(dict(parent=cid)), realm)
+            after_comp['mappers'] = sorted(kc.get_components(urlencode(dict(parent=cid)), realm), key=lambda x: x.get('name'))
```

Comment:
```suggestion
            after_comp['mappers'] = sorted(kc.get_components(urlencode(dict(parent=cid)), realm), key=lambda x: x.get('name') or '')
```
Otherwise you end up with `TypeError: '<' not supported between instances of 'NoneType' and 'str'`.

Same above.